api/api.pb.txt

```diff
@@ -787,6 +787,20 @@ file {
       type: TYPE_STRING
       json_name: "phpNamespace"
     }
+    field {
```

Comment:
This is from the recent protoc upgrade from https://github.com/docker/swarmkit/pull/2750/

Reply:
There's nothing to rebase against. The protoc upgrade change did not change this file, but it should have: regenerating the proto files after the upgrade will included these lines.

Reply:
(and sure, I could make it a separate PR, but is it really worth it?)